README.md

```diff
@@ -116,6 +116,7 @@ The API serves the following endpoints:
 *Related to checks:*
 - `GET` on `/api/checks/latest/?url={url}&resource_id={resource_id}` to get the latest check for a given URL and/or `resource_id`
 - `GET` on `/api/checks/all/?url={url}&resource_id={resource_id}` to get all checks for a given URL and/or `resource_id`
+- `GET` on `/api/checks/aggregate/?group_by={column}&created_at={date}` to get checks occurences grouped by a `column` for a specific `date`
```

Comment:
Not really related to this PR, but realizing now it would be better to write the URLs with GET parameters without trailing slashes in the documentation (like `/api/checks/aggregate?group_by={column}&created_at={date}`) since now the API supports both, and since GET parameters generally don't follow any trailing slashes.
Can make another PR to take care of that later.

Reply:
:+1: Done in https://github.com/datagouv/hydra/pull/167/commits/4e81f31f43177531ed5ec99451a1a0c97165eb6f.

Reply:
Perfect, thanks! There is a little typo, just commented